views/index.ejs

```diff
@@ -17,6 +17,7 @@
       <option value="">All Global Exposures</option>
       <option value="?exposure=Window">Window</option>
       <option value="?exposure=Worker">Worker</option>
```

Comment:
Since we're here, we might as well specify this is a dedicated worker?

```suggestion
      <option value="?exposure=Worker">Dedicated Worker</option>
```